io.github.Q1CHENL.fig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "io.github.Q1CHENL.fig",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "fig",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-run/gvfsd",
+    "--filesystem=home"
+  ],
+  "cleanup": [
+      "/include", 
+      "/lib/pkgconfig", 
+      "*.la", 
+      "*.a"
+  ],
+  "modules": [
+    {
+      "name": "python3-setuptools",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install . --prefix=${FLATPAK_DEST} --no-deps --ignore-installed"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/ed/22/a438e0caa4576f8c383fa4d35f1cc01655a46c75be358960d815bfbb12bd/setuptools-75.3.0.tar.gz",
+          "sha256": "fba5dd4d766e97be1b1681d98712680ae8f2f26d7881245f2ce9e40714f1a686"
+        }
+      ]
+    },
+    {
+      "name": "python3-pillow",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install . --prefix=${FLATPAK_DEST} --no-deps --ignore-installed --no-build-isolation"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/a5/26/0d95c04c868f6bdb0c447e3ee2de5564411845e36a858cfd63766bc7b563/pillow-11.0.0.tar.gz",
+          "sha256": "72bacbaf24ac003fea9bff9837d1eedb6088758d41e100c1552930151f677739"
+        }
+      ]
+    },
+    {
+      "name": "fig",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Q1CHENL/fig.git"
```

Comment:
I said this needs a tag / commit. It wasn't a suggestion.

Reply:
yes so I created a tag v1.0.0. Should also I put "tag": "v1.0.0" here? I looked at the [official demo](https://github.com/flatpak/qt-flatpak-demo/blob/master/org.flatpak.qtdemo.json) and [official doc](https://docs.flatpak.org/en/latest/manifests.html) they don't have that attribute.

Reply:
Ok so looked at other PRs i guess I indeed have to put it here

Reply:
why not reading the documentation instead? It's pretty clear.